source/openapi-admin-v3.yaml

```diff
@@ -4734,152 +4728,235 @@ components:
-            unordered:
-              type: boolean
-              description: |-
-                **Only Available for Database Triggers** -- If ``true``,
-                event ordering is disabled and this trigger can process
-                events in parallel. If ``false``, event ordering is
-                enabled and the trigger executes serially.
+        function_name:
+          type: string
+          description: The name of the function that the trigger calls when it fires.
         event_processors:
           type: object
           description: |-
-            An object where each field name is an event processor ID and each value
-            is an object that configures its corresponding event processor. The
-            following event processors are supported: `AWS_EVENTBRIDGE` For an
-            example configuration object, see [Send Trigger Events to AWS
+            An object where each field name is an event processor ID and
+            each value is an object that configures its corresponding
+            event processor. For an example configuration object, see
+            [Send Trigger Events to AWS
             EventBridge](https://www.mongodb.com/docs/realm/triggers/examples/send-events-aws-eventbridge#std-label-event_processor_example).
+          properties:
+            AWS_EVENTBRIDGE:
+              type: object
+              properties:
+                type:
+                  type: string
+                  enum: ["AWS_EVENTBRIDGE"]
+                config:
+                  type: object
+                  properties:
+                    account_id:
+                      type: string
+                      description: An AWS Account ID.
+                    region:
+                      type: string
+                      description: An AWS region.
+    DatabaseTrigger:
+      example:
+        name: onNewEmployee
+        type: DATABASE
+        function_name: handleNewEmployeeEvent
+        config:
+          operation_types:
+            - INSERT
+          database: HR
+          collection: employees
+          service_id: 5adeb649b8b998486770ae7c
+          match: {}
+          full_document: true
+      allOf:
+        - $ref: "#/components/schemas/BaseTrigger"
+        - properties:
+            type:
+              type: string
+              description: The trigger type. For database triggers, this value is always `"DATABASE"`.
+              enum: ["DATABASE"]
+            config:
+              required:
+                - service_id
+                - database
+                - collection
+                - operation_types
+                - full_document
+                - full_document_before_change
+                - unordered
+                - match
+                - project
+              properties:
+                service_id:
+                  type: string
+                  description: The _id value of a linked MongoDB data source.
```

Comment:
Is it possible to link to the endpoints or docs you can use to get this information? This stood out to me when thinking about how to get "the _id value of a linked MongoDB data source" but that might also apply to other things in this list.

Reply:
Yeah I can add that here. Probably a larger effort to do this more generally across the API.